crates/secret-service-proto/src/v1/wire.rs

```diff
@@ -576,3 +494,27 @@ impl TryFrom<SerializableTaprootWitness> for TaprootWitness {
         }
     }
 }
+
+#[derive(Debug, Clone, Copy, Archive, Serialize, Deserialize)]
+pub enum SignerTarget {
+    General,
+    Stakechain,
+    Musig2,
+}
+
+#[derive(Debug, Clone, Copy, Archive, Serialize, Deserialize)]
+pub struct WotsKeySpecifier {
+    /// [`Txid`](bitcoin::Txid) that the WOTS key is derived from.
+    pub txid: [u8; 32],
```

Comment:
I think you might be able to replace 32 with this: https://docs.rs/secp256k1/0.29.1/secp256k1/hashes/sha256d/struct.Hash.html#associatedconstant.LEN